gui/OpenDungeonsIcons.imageset

```diff
@@ -23,6 +23,8 @@
     <Image height="60" name="PortalButton" width="60" xPos="386" yPos="194" />
     <Image height="60" name="SummonWorkerButton" width="60" xPos="2" yPos="322" />
     <Image height="60" name="CallToWarButton" width="60" xPos="66" yPos="322" />
+    <Image height="60" name="CreatureHealButton" width="60" xPos="192" yPos="194" />
```

Comment:
Yep, see  "Create the shadow". Feel free to upgrade the page if it feels vague or inadequate to anything.